packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-wrapper.tsx

```diff
@@ -71,29 +70,29 @@ const JurisdictionModalContentWrapper = observer(({ openPasswordModal }: TJurisd
 
     const financial_available_accounts = trading_platform_available_accounts.filter(
         available_account =>
-            available_account.market_type === MARKET_TYPE.FINANCIAL &&
+            available_account.market_type === 'financial' &&
             (show_eu_related_content
-                ? available_account.shortcode === JURISDICTION.MALTA_INVEST
-                : available_account.shortcode !== JURISDICTION.MALTA_INVEST)
+                ? available_account.shortcode === 'maltainvest'
+                : available_account.shortcode !== 'maltainvest')
```

Comment:
Why did you make changes here ?

Reply:
I think she pull from master and the changes are reverted.
Need to check other files too.